Jenkinsfile

```
@@ -58,7 +58,7 @@ pipeline {
                     if (body.contains("[test cassandra]") || title.contains("cassandra:")) {
                         env.testProvider = "cassandra"
                     } else if (body.contains("[test ceph]") || title.contains("ceph:")) {
-                        env.testProvider = "ceph"
+                        env.shouldBuild = "false"
```

if [test full] is included along with the ceph prefix, we still need to run the build. Perhaps we need this:
```suggestion
                        if (body.contains("[test full]")) {
                          env.testProvider = "ceph"
                        } else {
                          env.shouldBuild = "false"
                        }
```

---

Done.